icons/protocols-dot-io.svg

```diff
@@ -1 +1 @@
6 11.0067 11.0067 0 00-.42 3.5843c.052 1.007-.187 1.8881-.6 2.7972C.8577 11.0239.2696 12.967.0306 15.0042c-.136 1.1641.169 2.1502 1.13 2.8712.4101.308.8341.603 1.2802.855 1.241.7021 2.5731 1.1911 3.9242 1.6332a.825.825 0 01.297.162c1.12 1.033 2.4232 1.75 3.8462 2.2761 1.0111.373 2.0052.364 3.0152-.01 1.0571-.391 2.0892-.853 2.9442-1.576.76-.6421 1.6121-1.0112 2.5272-1.3342 1.307-.461 2.5661-1.036 3.6902-1.871.934-.6951 1.413-1.5962 1.298-2.7783-.023-.238-.04-.476-.076-.712-.3-1.9421-.888-3.8022-1.705-5.5803-.356-.776-.658-1.5241-.441-2.4032-.075-.856-.128-1.726-.233-2.5881-.083-.687-.24-1.3641-.5741-1.9822-.442-.821-1.126-1.164-2.0371-1-.32.057-.642.144-.94.273-1.6102.698-2.9043 1.7991-3.9493 3.1972-.101.137-.195.184-.369.182a190.3816 190.3816 0 00-3.3283-.002c-.144.001-.241-.014-.334-.144-.623-.877-1.389-1.6141-2.2561-2.2462-.735-.536-1.5101-1-2.4062-1.22a2.6012 2.6012 0 00-.783-.08zm.096 1.152c.087-.002.183.012.292.04.34.085.685.194.996.357 1.4302.7461 2.5962 1.7922 3.4433 3.1733.088.144.178.186.338.178.904-.042 1.8061-.097 2.7102-.103.62-.004 1.24.06 1.86.103.163.012.2501-.04.3361-.182.855-1.3871 2.0241-2.4402 3.4692-3.1812.304-.156.64-.265.973-.348.4311-.107.7021.006.8631.418.188.484.352.99.43 1.5.123.8151.166 1.6422.226 2.2812-.037.648-.07 1.108-.086 1.569a.68.68 0 00.08.3221 21.8203 21.8203 0 012.1742 6.3774c.073.416.077.85.064 1.273-.013.4811-.324.8121-.673 1.1001-.8731.72-1.8882 1.1881-2.9262 1.6021-.714.284-1.4521.51-2.1742.775-.15.055-.3.1481-.418.2581-1.224 1.13-2.6252 1.9401-4.2603 2.2851-.37.078-.804.014-1.174-.098-1.4731-.445-2.7882-1.18-3.8922-2.2701a.692.692 0 00-.272-.148c-1.3501-.431-2.6842-.905-3.9313-1.5881-.35-.19-.676-.42-1.014-.632-.8101-.51-1.0261-1.2671-.8931-2.1662.355-2.3871 1.131-4.6323 2.2351-6.7724a.745.745 0 00.07-.375c-.106-1.5221-.064-3.0342.293-4.5253.064-.265.162-.528.281-.774.146-.299.32-.442.58-.449zm9.2216 6.7204c-.354.04-.539.263-.719.72-.032.0831-.06.1671-.0821.2521-.196.782-.121 1.5631-.025 2.3522.108.89.498 1.492 1.367 1.832 1.3792.5401 2.1102 1.6452 2.4422 3.0473.08.343.148.688.228 1.068.39-.138.764-.265 1.1351-.404.998-.373 1.9721-.794 2.8632-1.39.15-.1001.224-.2121.195-.3931-.091-.586-.142-1.181-.273-1.7581-.409-1.8061-1.407-3.1782-3.0802-4.0183-.61-.305-1.222-.614-1.8611-.845a9.5856 9.5856 0 00-1.7741-.444 1.5861 1.5861 0 00-.416-.02zm-4.2073.01a2.2601 2.2601 0 00-.264.035C7.8262 9.1649 6.4 9.81 5.098 10.737c-.489.348-.91.766-1.225 1.282-.7271 1.1931-1.0731 2.5002-1.1521 3.8863-.01.178.053.288.199.364.722.375 1.433.777 2.1721 1.117.583.268 1.2.4621 1.8241.6971.083-.39.15-.737.23-1.08.331-1.4021 1.0731-2.4802 2.4362-3.0602.05-.021.093-.052.143-.069.723-.249 1.069-.77 1.185-1.5161.143-.919.259-1.8291-.03-2.7362-.221-.7-.494-.872-1.22-.812zm2.5942 7.5025c-.9601.045-1.6531.137-2.2742.505-.426.254-.559.555-.396.9791.366.952.999 1.6901 1.8901 2.1901.33.184.685.193 1.02.011.9181-.499 1.5542-1.252 1.9282-2.2201.14-.365.044-.626-.274-.864-.147-.11-.312-.219-.486-.266-.544-.146-1.0961-.263-1.4081-.335z"/></svg>
```

Comment:
Icon looks perfect! 🎉
Could you try and get it down to 3 decimals please?

Reply:
@adamrusted Done 👍🏼